wolox-books/src/app/screens/Login/components/login/login.component.scss

```diff
@@ -41,21 +41,23 @@
   width: 100%;
 }
 
-.sign-up-btn {
+.login-btn {
```

Comment:
If you are using the same styles for the `login-btn` and the `sign-up-btn`, just create a `.primary-btn` class in the common styles and use that 😄

Reply:
I'm not using the same styles for those classes, actually they differ in certain things, that's why they are separated. The common thing that they share, is the `%generic-btn` styles, that is being imported from the generic styles file, and as you can see, both of them have the `@extend %generic-btn;` sentence. There is where I'm sharing the styles that are common to both. Probably you thought that because of the diffing, the diffing is appearing like that because I was using the wrong name clase for the buttons: the button that should've been login was with the sign-up class, and viceversa, that's why the diffing is confusing.